README.md

```diff
@@ -1,2 +1,42 @@
-# LeaderBoard
-Leaderboard applicaton | JavaScript
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> Leader board  application using Js/webpack
+
+
+## Built With
+
+- Js
+- web pack
+
+## Live Demo 
+> 
+
+
+## Getting Started
+This is a simple html/css and js  project.
+
+
+## Authors
+
+👤 **Alazar Yilma**
+
+- GitHub: [@Agetuni](https://github.com/Agetuni)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aleazar-yilma-b614b6174/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- Please, kindly update the link to the issues page so it links to issues page of your project on Github to make it easy for users to navigate to the issues page.